src/js/keyboardInput.js

```diff
@@ -198,6 +207,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     gpii.firstDiscovery.keyboardInput.registerKeypressListener = function (that, input, keymap) {
         input.keypress(function (e) {
             e.preventDefault();
+            e.stopPropagation();
```

Comment:
Using return false here feels a little 'magical' to me. I like the explicitness of using e.preventDefault() and e.stopPropagation(). But I am totally open to making the change, particularly if using return false is more idiomatic within our code base.